<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="logText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>1041</width>
      <height>171</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Log...</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="deviceLabel">
       <property name="text">
        <string>Device:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="deviceComboBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLineEdit" name="inputFilePath">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>File path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="chooseFileButton">
       <property name="text">
        <string>Choose file</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>271</width>
      <height>373</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="autostraightenCheckBox">
       <property name="text">
        <string>Autostraighten</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="breakAfterEachCheckBox">
       <property name="text">
        <string>Break pages after each source page</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="colorOutputCheckBox">
       <property name="text">
        <string>Color output</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="landscapeCheckBox">
       <property name="text">
        <string>Rotate output to landscape</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="nativePDFOutputCheckBox">
       <property name="text">
        <string>Native PDF output</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="rightToLeftCheckBox">
       <property name="text">
        <string>Right to left text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="generateMarkedUpSourceCheckBox">
       <property name="text">
        <string>Generate marked-up source</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="reFlowTextCheckBox">
       <property name="text">
        <string>Re-flow text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="eraseVerticalLinesCheckBox">
       <property name="text">
        <string>Erase vertical lines</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>281</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </widget>
   <widget class="QPushButton" name="convertFileButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1048</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
